### ISSUE TITLE
Fetch OPAC URL templates from source file

### ISSUE THREAD

**Issue by acka47** [hbz/nwbib#387]
Currently, we replicate the file https://github.com/hbz/link-templates/blob/master/isil2opac_hbzid.json both in this repo and in the lobid-resources repo. As this file needs to be updated quite often, this is impractical. In the future, OPAC URL templates should directly be fetched from GitHub at https://raw.githubusercontent.com/hbz/link-templates/master/isil2opac_hbzid.json. Corresponding issue in lobid-resources is https://github.com/hbz/lobid-resources/issues/397.

**Comment by acka47** [comment]
@fsteeg Isn't this already implemented, as well as https://github.com/hbz/lobid-resources/issues/397?

**Comment by fsteeg** [comment]
Off the top of my head I'd say it's implemented in resources, but not here in NWBib. Will investigate.

**Comment by TobiasNx** [comment]
> Wird unnötig durch #628
> 
> Kann ich schließen @fsteeg ?

@acka47 wir können das doch schließen, da wir jetzt für ALMA die OPAC Links anders erzeugen

**Comment by fsteeg** [comment]
> wir können das doch schließen, da wir jetzt für ALMA die OPAC Links anders erzeugen

Ja, im Prinzip schon, allerdings könnte dann das Template-Zeug auch mal weg.
